app/locales/pt-BR.json

```diff
@@ -0,0 +1,118 @@
+{
+  "label": {
+    "about_title": "Sobre",
+    "authorities_add_button_label": "Adicionar uma fonte confiável",
+    "authorities_add_url": "Adicionar uma autoridade via URL",
+    "authorities_desc": "Escolha autoridades de saúde confiáveis ​​em sua área para obter dados de exposição. Selecione um nome no registro global, ou digite o endereço da web fornecido por uma autoridade que implementou o Safe Paths.",
+    "authorities_input_placeholder": "Cole a URL aqui",
+    "authorities_no_sources": "Nenhuma fonte de dados ainda",
+    "authorities_removal_alert_cancel": "Cancelar",
+    "authorities_removal_alert_desc": "Você tem certeza que deseja remover this authority data source?",
+    "authorities_removal_alert_proceed": "Continuar",
+    "authorities_removal_alert_title": "Remover autoridade",
+    "authorities_title": "Fontes Confiáveis",
+    "choose_provider_subtitle": "Para ser informado de exposição você deverá contatar uma agência de saúde.",
+    "choose_provider_title": "Escolha a autoridade de saúde",
+    "commitment": "Comprometimento",
+    "commitment_para": "Safe Paths armazena e checa sua interação com as pessoas usando sua localização. Os seus dados NUNCA sairão dos seu celular sem o seu consentimento.",
+    "default_news_site_name": "Novidades do Safe Paths",
+    "event_history_subtitle": "Entender sua exposição pessoal baseada na informação compartilhada com as autoridades de saúde.",
```

Comment:
compartilhadas pelas autoridades de saúde instead of compartilhada com as autoridades de saúde?

in English, it is shared by health authorities.

Reply:
Who is sharing the information? Is the user or the health authorities?

Reply:
"event_history_subtitle": "Understand your personal exposure based on information shared by health authorities.",

The information is shared by the health authorities with you. Based on this information, you can understand how you are exposed.